deployments/security-groups.nix

```diff
@@ -119,6 +120,15 @@ with import ../lib.nix;
                 sourceIp = "0.0.0.0/0";
               }];
             };
+            "allow-to-faucet-${region}" = {
+              inherit region accessKeyId;
+              description = "Access Cardano Faucet";
+              rules = [{
+                protocol = "tcp";
+                fromPort = 80; toPort = 80;
```

Comment:
should be 443/ssl instead of allowing insecure 80 traffic

Reply:
OK I will add 443 and leave 80 to allow redirecting http -> https.